docs/user/tutorials/01-10-external-authorization-provider.md

```diff
@@ -1,91 +1,80 @@
+The Istio CR allows configuring external authorization providers that operate over HTTP. To set up the oauth2-proxy external authorization provider in the Istio CR and expose your workload using an Istio VirtualService, follow these steps.
 
-    <!-- tabs:start -->
-    #### **Custom Domain**
+## Procedure
+1. Export the following values as environment variables:
 
     ```bash
-    export DOMAIN_TO_EXPOSE_WORKLOADS={DOMAIN_NAME}
-    export GATEWAY=$NAMESPACE/httpbin-gateway
+    export WORKLOAD_NAME={WORKLOAD_NAME}
+    export NAMESPACE={WORKLOAD_NAMESPACE}
+    export DOMAIN_TO_EXPOSE_WORKLOADS={YOUR_DOMAIN}
+    export GATEWAY={YOUR_GATEWAY}
     ```
-    #### **Kyma Domain**
 
-    ```bash
-    export DOMAIN_TO_EXPOSE_WORKLOADS={KYMA_DOMAIN_NAME}
-    export GATEWAY=kyma-system/kyma-gateway
-    ```
-    <!-- tabs:end -->
+    Option | Description |
+    ---------|----------|
+    WORKLOAD_NAME | The name of your workload. |
+    NAMESPACE | The name of your workload's namespace. |
+    DOMAIN_TO_EXPOSE_WORKLOADS | The domain that should be used to expose the workloads. |
+    GATEWAY | The Gateway that the VirtualService should use to route traffic to the workload. |
 
-## Steps
+2. Create a VirtualService to expose the workload:
 
-### Create a Workload
+    ```bash
+    cat <<EOF | kubectl apply -f -
+    apiVersion: networking.istio.io/v1alpha3
+    kind: VirtualService
+    metadata:
+      name: ext-authz
+      namespace: $NAMESPACE
+    spec:
+      hosts:
+      - "$WORKLOAD_NAME.$DOMAIN_TO_EXPOSE_WORKLOADS"
+      gateways:
+      - $GATEWAY
+      http:
+      - match:
+        - uri:
+            prefix: /
+        route:
+        - destination:
+            port:
+              number: 80
+            host: $WORKLOAD_NAME.$NAMESPACE.svc.cluster.local
+    EOF
+    ```
 
-1. Export the name of the namespace in which you want to deploy the HTTPBin Service:
+3. Export the following configuration values as environment variables:
 
-    ```
-    export NAMESPACE={NAMESPACE_NAME}
+    ```bash
+    export CLIENT_ID={YOUR_CLIENT_ID}
+    export CLIENT_SECRET={YOUR_CLIENT_SECRET}
+    export COOKIE_SECRET={YOUR_COOKIE_SECRET}
+    export OIDC_ISSUER_URL={YOUR_OIDC_ISSUER_URL}
     ```
 
-2. Create a namespace with Istio injection enabled and deploy the HTTPBin Service:
+    Option | Description |
+    ---------|----------|
+    CLIENT_ID | 	The unique identifier for the client application that is registered with the external authorizer. |
+    CLIENT_SECRET | A secret key known only to the client and the external authorizer. It is used to authenticate the client when communicating with the external authorizer. To generate this value, you can use the command `openssl rand -base64 32 | head -c 32 | base64.` |
+    CLIENT_COOKIE | A secret key used to sign and encrypt the cookies that are used for session management and user authentication. To generate this value, you can use the command `openssl rand -base64 32 | head -c 32 | base64`. |
```

Comment:
The pipe signs in the commands interfere with the table formatting:
![Screenshot 2024-10-24 at 19 24 42](https://github.com/user-attachments/assets/7c06eee5-f648-4232-874c-9e8fdd83aac3)

Reply:
I replaced it with HTML syntax